xml/System.Runtime.InteropServices/CallingConvention.xml

```diff
@@ -197,7 +197,7 @@
       </ReturnValue>
       <MemberValue>3</MemberValue>
       <Docs>
-        <summary>The callee cleans the stack.</summary>
+        <summary>The caller cleans the stack.</summary>
```

Comment:
This is incorrect. In the stdcall calling convention the [callee cleans up the stack](https://learn.microsoft.com/cpp/cpp/stdcall).